macros/ton_muons_vertical.config.mac

```diff
@@ -29,13 +28,22 @@
 /Geometry/NextTonScale/gas            enrichedXe
 
 ### GENERATOR
-/Generator/MuonAngleGenerator/region MUONS
-/Generator/MuonAngleGenerator/min_energy 200 GeV
-/Generator/MuonAngleGenerator/max_energy 250 GeV
-/Generator/MuonAngleGenerator/angles_on false
+/Generator/MuonGenerator/region MUONS
+/Generator/MuonGenerator/min_energy 200 GeV
+/Generator/MuonGenerator/max_energy 250 GeV
+
+# Use angular distribution simulated/measured at LSC
+/Generator/MuonGenerator/use_lsc_dist false
+
+# User specified muon direction. Must set use_lsc_dist to false for this to work
+/Generator/MuonGenerator/user_dir 0 -1 0
+
+# For coordinate system transformation, do not edit
+/Generator/MuonGenerator/azimuth_rotation 150 deg
 
 ### ACTIONS
 /Actions/DefaultEventAction/min_energy 0.01 MeV
+#/Actions/MuonsEventAction/stringHist MuonsDistribution.root
```

Comment:
I would use `csv` as a format in the example, to make it as general as possible (so that one doesn't need to install ROOT to read it). The same applies to `NEXT100_muons.config` and `DEMO_muons.config`.